p5js/ImageClassification/ImageClassification_Teachable-Machine/index.html

```diff
@@ -3,17 +3,16 @@
 <head>
   <meta charset="UTF-8">
   <title>Webcam Image Classification using a pre-trained customized model and p5.js</title>
-
   <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.9.0/p5.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.9.0/addons/p5.dom.min.js"></script>
-
   <script src="http://localhost:8080/ml5.js" type="text/javascript"></script>
```

Comment:
just making a note that we will update this URL with a big `find + replace` when it goes into the `release` branch.

Reply:
Also, for when this goes live we should retrain the example model so that we have one with the new TM url.